app/utils/camera_discovery.py

```
@@ -0,0 +1,107 @@
+import uuid
+import psutil
+import logging
+import xml.etree.ElementTree as ET
+from urllib.parse import urlparse
+from ipaddress import ip_network
+from .screenshots import is_port_open
+
+
+def _local_subnets():
+    subnets = []
+    for iface, addrs in psutil.net_if_addrs().items():
```

Ruff (0.11.9)

Loop control variable `iface` not used within loop body

Rename unused `iface` to `_iface`

(B007)